packages/uswds-core/src/styles/mixins/general/icon.scss

```diff
@@ -77,7 +77,9 @@
 // ($base, $variant, $variant-alt, $bg)
 // the mixin will pick IE11-compatible svgs named
 // [base]-[variant].svg based on the specified background-color
-
+//
+// @param {Map} - $icon-object - name, svg-height, svg-width, height, container-height, container-width, color, color-variant, color-hover, rotate, path, background-p
```

Comment:
Added Sassdoc style comments to document the types of things `$icon-object` accepts in this mixin.